src/OpenTK.Mathematics/Matrix/Matrix3x2.cs

```diff
@@ -71,15 +72,11 @@ public Matrix3x2(Vector2 row0, Vector2 row1, Vector2 row2)
         /// <param name="m11">Second item of the second row of the matrix.</param>
         /// <param name="m20">First item of the third row of the matrix.</param>
         /// <param name="m21">Second item of the third row of the matrix.</param>
-        public Matrix3x2
-        (
-            float m00,
-            float m01,
-            float m10,
-            float m11,
-            float m20,
-            float m21
-        )
+        [SuppressMessage("ReSharper", "SA1117", Justification = "For better readability of Matrix struct.")]
+        public Matrix3x2(
```

Comment:
I like the idea of these constructor "shapes", but OpenTK uses the "Allman" brace style, which means you should move the parentheses like this:
```csharp
public Matrix2x3
(
    // arguments
    // more arguments
)
```